--- a/src/fabric.c
+++ b/src/fabric.c
@@ -1252,13 +1252,13 @@ struct fi_info *DEFAULT_SYMVER_PRE(fi_dupinfo)(const struct fi_info *info)
                dup->ep_attr = mem_dup(info->ep_attr, sizeof(*info->ep_attr));
                if (dup->ep_attr == NULL)
                        goto fail;
-               if (info->ep_attr->auth_key != NULL) {
-                       dup->ep_attr->auth_key =
-                               mem_dup(info->ep_attr->auth_key,
-                                       info->ep_attr->auth_key_size);
-                       if (dup->ep_attr->auth_key == NULL)
-                               goto fail;
-               }
+               //if (info->ep_attr->auth_key != NULL) {
+               //      dup->ep_attr->auth_key =
+               //              mem_dup(info->ep_attr->auth_key,
+               //                      info->ep_attr->auth_key_size);
+               //      if (dup->ep_attr->auth_key == NULL)
+               //              goto fail;
+               //}
        }
        if (info->domain_attr) {
                dup->domain_attr = mem_dup(info->domain_attr,
@@ -1272,13 +1272,13 @@ struct fi_info *DEFAULT_SYMVER_PRE(fi_dupinfo)(const struct fi_info *info)
                        if (dup->domain_attr->name == NULL)
                                goto fail;
                }
-               if (info->domain_attr->auth_key != NULL) {
-                       dup->domain_attr->auth_key =
-                               mem_dup(info->domain_attr->auth_key,
-                                       info->domain_attr->auth_key_size);
-                       if (dup->domain_attr->auth_key == NULL)
-                               goto fail;
-               }
+               //if (info->domain_attr->auth_key != NULL) {
+               //      dup->domain_attr->auth_key =
+               //              mem_dup(info->domain_attr->auth_key,
+               //                      info->domain_attr->auth_key_size);
+               //      if (dup->domain_attr->auth_key == NULL)
+               //              goto fail;
+               //}
        }
        if (info->fabric_attr) {
                dup->fabric_attr = mem_dup(info->fabric_attr,
